external-builds/pytorch/build_prod_wheels.py

```diff
@@ -535,6 +548,7 @@ def do_build_pytorch(
     *,
     triton_requirement: str | None,
 ):
+    root_dir = get_rocm_path("root")
```

Comment:
Move this to where it is used. Unless many other lines in the function are using the variable it should be kept near the specific code that uses, with minimal scope.

You can move it to here:
```diff
        # libdrm/drm.h.
+       root_dir = get_rocm_path("root")
+       sysdeps_dir = root_dir / "lib" / "rocm_sysdeps"
-       sysdeps_dir = get_rocm_path("root") / "lib" / "rocm_sysdeps"
        assert sysdeps_dir.exists(), f"No sysdeps directory found: {sysdeps_dir}"
        add_env_compiler_flags(env, "CXXFLAGS", f"-I{sysdeps_dir / 'include'}")
        # Add correct include path for roctracer.h (for Kineto)
        rocm_include = root_dir / "include"
```